cf-atcp-contact/config/example.env

```diff
@@ -0,0 +1,20 @@
+# This is an example of the env vars needed by the function.
```

Comment:
I switched to using env vars instead of a config file. Some advantages:

* Smaller size. No need to include the `typesafe config` dependency.
* Should be simple to use (just source the file in your shell).
* Works nicely with `gcloud functions deploy`.
* You can see the env vars in gcloud console website, so everything about a function is in one place.